### PR TITLE
[21.05] Fix edit field reactivity in libraries

### PR DESCRIPTION
**Pull request by OlegZharkov**
PR fixes edit field reactivity in Libraries. 
Fixes https://github.com/galaxyproject/galaxy/issues/12005

## How to test the changes?
please follow
https://github.com/galaxyproject/galaxy/issues/12005

## License
- [x] I agree to license these contributions under [Galaxy's current license](https://github.com/galaxyproject/galaxy/blob/dev/LICENSE.txt).
- [x] I agree to allow the Galaxy committers to license these and all my past contributions to the core galaxy codebase under the [MIT license](https://opensource.org/licenses/MIT). If this condition is an issue, uncheck and just let us know why with an e-mail to galaxy-committers@lists.galaxyproject.org.

### CODE REVIEW

File: client/src/components/Libraries/LibraryEditField.vue [hunk 1]
```diff
@@ -3,7 +3,7 @@
         <div class="text-field">
             <!-- edit mode -->
             <div v-if="isEditMode">
-                <textarea class="form-control" v-model="textField" rows="3" />
+                <b-form-textarea class="form-control" :value="text" @change="updateValue" rows="3" no-resize />
```
Comment (mvdbeek) [inline client/src/components/Libraries/LibraryEditField.vue:6]: did v-model not work here ?

Comment (mvdbeek) [reply client/src/components/Libraries/LibraryEditField.vue:6]: Oh I see, in `updateValue` you probably want to emit the new value back to the parent, where the parent component should fire a request and update the value that gets passed in as a prop

Comment (OlegZharkov) [reply client/src/components/Libraries/LibraryEditField.vue:6]: When we change the page, the `text` prop is getting updated. 
However `textField` is assigned to `text` only on creation https://github.com/galaxyproject/galaxy/blob/dev/client/src/components/Libraries/LibraryEditField.vue#L58
so I made `:value` to be equal prop and change `textField` only on `change` event

Comment (OlegZharkov) [reply client/src/components/Libraries/LibraryEditField.vue:6]: > Oh I see, in `updateValue` you probably want to emit the new value back to the parent, where the parent component should fire a request and update the value that gets passed in as a prop

:+1:

**Comment by davelopez** [comment]
Thanks @OlegZharkov it works now!

File: client/src/components/Libraries/LibraryEditField.vue (hunk 1 above)
Comment (mvdbeek) [reply client/src/components/Libraries/LibraryEditField.vue:6]: Are you going to do this then ? That means you can replace https://github.com/mvdbeek/galaxy/blob/06819532f1a317ee91aed1936c788ee79f911aba/client/src/components/Libraries/LibrariesList.vue#L248-L249. I also noticed that the name field is editable, but it has no handler attached.

**Pull request merged**